.gitignore

```diff
@@ -6,3 +6,5 @@ __pycache__
 .idea/
 .coverage*
 .mypy_cache
+notebooks/.DS_Store
```

Comment:
This should be a separate PR, as Brian mentioned.

Reply:
Not quite sure I understand why? .DS_Store is a binary file used on Mac to keep track of the Finder UI. It changes when you so much as blink and regenerates itself when deleted. If not in .gitignore, it'll keep regenerating itself and try to add itself to every commit I make, so this was for my own sanity and to keep the remote repo free of unneeded files.

Reply:
You have to `git add /specific/file` when it's not in the ignore file.  Don't always use `git add .`.